lib/media.js

```diff
@@ -97,10 +97,16 @@ exports.setTrack = function (item, callback) {
       callback(err)
       return
     }
-    var duration = 0
+    var duration;
```

Comment:
not sure why removing initial value?
also please be consistent without semicolons.